markdown/manual.md

```diff
@@ -1,220 +1,251 @@
+
+Should you encounter a situation that suggests a conflict of interest, please refer the situation the Executive Director, who will in turn seek a determination from the Board of Directors as to whether a conflict exists, and if it exists, whether it is an allowable conflict.
 
 ### Professional Conduct
-The company adheres to the highest legal and ethical standards applicable in our business.  The company's business is conducted in strict observance of both the letter and spirit of all applicable laws and the integrity of each employee is of utmost importance.
+Our organization adheres to the highest legal and ethical standards applicable in our line of work. The organization's business is conducted in strict observance of both the letter and spirit of all applicable laws and the integrity of each employee is of utmost importance.
 
 ### Contact with the Media
-All media inquiries regarding the company and its operations must be referred to the Executive Director.  Only the Executive Director or Chairman of the Board is authorized to make or approve public statements on behalf of the company.
+We strive to balance the need for intentionality in our organizational communications with each employee's fundamental free speech rights and the import role of media in our society.
+
+It is our policy that media inquiries directly pertaining to the organization, its employees, products, positions, or related operations be referred to the Executive Director.
 
+We also encourage individual employees to speak to members of the media about any subject matter of their choosing, including any subject matter in which the organization may be involved. We simply ask that you present your views as "personal" and not of the organization (unless authorized to do so).
 
 ## Staff & Boards
 
 ### Staff
 * Seamus Kraft, Co-Founder and Executive Director
-* Meag Doherty, Chief of Staff
-* Seth Etter, Senior Developer
-* Tanner Doshier, Senior Developer
-* Mary Kate Mezzetti, Intern
+* Patrick Bateman, Deputy Executive Director & Chief of Staff
+* Aaron Ogle, Director of Product
+* Seth Etter, Senior Engineer
+* Tanner Doshier, Senior Engineer
+* Kunal Kothari, Fellow
 
 ### Board of Directors
-* Darrell Issa, Chairman
-* Seamus Kraft, President and Vice-Chairman
+* Phaedra Chrousos, Chair
+* Laurent Crenshaw, Vice Chair
+* Seamus Kraft, Secretary
 * Rich Hirshberg, Treasurer
-* James Lacy, Counsel
-* Tom Davis, Member
-* Abhi Nemani, Member
-* Larry Brady, Member
+* Michelle Lee
+* Aaron Snow
 
 ### Board of Advisers
 * Scout Addis, Design
-* Dr. Anne Washington, Legal Data and Informatics
-* Joe Trippi, Political Campaigns
-* Karien Bezuidenhout, Shuttleworth Foundation
-* Lanham Napier, BuildGroup
+* Karien Bezuidenhout, Internal Growth
+* Dr. Anne Washington, Open Legal Data
 * Aaron Bartnick
```

Comment:
Just noticed Aaron Bartnick is not listed on [the website](https://opengovfoundation.github.io/opengovfoundation.org/#team). Not sure which is out of sync.